examples/visual-tests/amp-sidebar/amp-sidebar-toolbar-ol.amp.html

```diff
@@ -44,6 +41,11 @@
       width: 50vw;
     }
 
+    .i-amphtml-sidebar-mask, amp-sidebar[side] {
```

Comment:
Please add a note mentioning that you're using invalid AMP (`.i-amphtml-`, `!important`) for the purposes of disabling animation for this test.

Reply:
I believe these lines are unnecessary, as Percy disabled all CSS animations: https://docs.percy.io/docs/animations#section-automatic-animation-freezing-in-percy

Have you tried manually running these test multiple times on a personal account and see if the animation causes any visual diffs?

Reply:
Good catch! Re-tested and safely reintroduced the animation styles. However the `transform: none` needs to stay and I left a comment accordingly.

Reply:
// cc @timhaines from Percy about not disabling `transform`s - is this a bug or a feature? :)